src/defs.cpp

```diff
@@ -63,7 +63,7 @@ PacketTimes *g_pPacketTimes = NULL;
 
 TicksTime g_lastTicks;
 
-const int MAX_FDS_NUM = os_get_max_active_fds_num();
+int MAX_FDS_NUM = 0;
```

Comment:
This is no more a const, so we must not use name capitalization:
```suggestion
int max_fds_num = 0;
```